newrelic-integration-e2e/main.go

```diff
@@ -32,23 +34,27 @@ func processCliArgs() (string, string, string, logrus.Level) {
 	if *specsPathPtr == "" {
 		logrus.Fatalf("missing required spec_path")
 	}
+	if *flagRootDirPtr == "" {
+		logrus.Fatalf("missing required root_dir")
```

Comment:
why the library isn't using a nil pointer to indicate a missing flag. 😖  :trollface: